### ISSUE TITLE
Is this fixed?

### ISSUE THREAD

**Issue by Bohirjon** [BurkusCat/MauiShellTabNavRepro#1]
Hi there! I was having the same issue. 
I want to navigate to a page with three tabs from the home page.

Have you found any solution for this by chance?

**Comment by BurkusCat** [comment]
No, I didn't. The conclusion I came to was that it doesn't appear to be possible in Shell [without workarounds/hacks](https://stackoverflow.com/questions/77017869/how-do-i-navigate-from-shellcontent-a-contentpage-to-a-tabbar-tabbedpage-in-net).

However, I did solve the problem by creating my own library, [Burkus.Mvvm.Maui](https://github.com/BurkusCat/Burkus.Mvvm.Maui). It's simple to setup and you use it instead of using Shell. Navigating to a TabbedPage from a ContentPage is trivial in it. I've just updated it to use .NET 8 today.

I'd encourage you to give a try and see if it makes your life easier :)
